libs/backend/package.json

```diff
@@ -0,0 +1,18 @@
+{
```

Comment:
rename `libs/backend` to `libs/backend-types` seems like only types will be stored here and shared between BE and FE